app/articles/views.py

```diff
@@ -118,6 +119,13 @@ class ArticleUnFavouriteView(generics.UpdateAPIView):
     renderer_classes = (JSONRenderer,)
 
 
+class ArticleStatsView(generics.ListCreateAPIView):
```

Comment:
What if I want statistics for a particular article not all?

Reply:
Making changes in the files to implement this